llvm/test/MC/GOFF/basic-goff-64.ll

```diff
@@ -0,0 +1,252 @@
+; RUN: llc %s -mtriple s390x-ibm-zos -filetype=obj -o - | od -v -Ax -tx1 | FileCheck --ignore-case %s
```

Comment:
Can we use obj2yaml for this type of tests?

Reply:
Not currently (obj2yaml doesn't support TXT records yet), but once that PR is in the intention is to augment the existing tests instead of relying on `od`.